drift_dev/lib/src/analyzer/dart/column_parser.dart

```diff
@@ -355,6 +359,17 @@ class ColumnParser {
       );
     }
 
+    if (foundFeatures.contains(const UniqueKey()) &&
```

Comment:
Could you add tests for the error cases similar to the one in say `test/analyzer/dart/column_references_test.dart`?